### ISSUE TITLE
Ignore node_modules by default

### ISSUE THREAD

**Issue by Digicrat** [apidoc/apidoc#937]
Unless I explicitly ignore node_modules with the -e parameter, I am getting a variety of errors when running apidoc.

At least when loading the apidoc configuration from package.json, it would be prudent to automatically add "node_modules" to the exclusions list.

**Comment by NicolasCARPi** [comment]
Why did you open 2 issues?

Closing as duplicate of #936

**Comment by Digicrat** [comment]
This one was a feature request (ignore node_modules folder by default when package.json is used), which is different from the bug in the other (emacs .#tmp and #tmp# filename forms causing  'file not found' error).  

For reference, the errors I saw from the node_modules not being ignored were different, and simply the result of some dependency I pulled in having invalid syntax (plus there being a lot of dependencies).  

(And I just realized github didn't actually mark this as a feature request when I selected that in opening the issue).
